### PR TITLE
Fix adding a schedule to a range use plan

### PR DESCRIPTION
**Pull request by jleach**
Fixed the db relation for plans and schedules and implemented the logic go add a schedule to a plan.

### CODE REVIEW

File: server/router/routes/plan.js [hunk 1]
```diff
@@ -43,44 +43,48 @@ const {
...
   EXCLUDED_PLAN_ATTR,
   STANDARD_PLAN_INCLUDE,
+  INCLUDE_ZONE_MODEL,
 } = dm;
 
 const router = new Router();
 
+const filterZonesOnUser = (user) => {
+  if (!user.isAdministrator()) {
+    return { INCLUDE_ZONE_MODEL, where: { userId: user.id } };
```
Comment (Kyubinhan) [inline server/router/routes/plan.js]: return { ...INCLUDE_ZONE_MODEL, where: { userId: user.id } };

Comment (jleach) [reply server/router/routes/plan.js]: 👍 I fin'd I'm copying `filterZonesOnUser` to a few places. Where do you think I should move it so I can `import` it? In DataManager.js or util.js?

Comment (jleach) [reply server/router/routes/plan.js]: I think it should go in DataModel with other data specific functions.

Comment (Kyubinhan) [reply server/router/routes/plan.js]: Yeah that should work!

**Pull request merged**